test/e2e/proxy_test.go

```diff
@@ -247,8 +253,8 @@ func testProxyServiceLifeCycle(ipFamily *corev1.IPFamily, ingressIPs []string, d
 	agentName, err := data.getAntreaPodOnNode(nodeName)
 	require.NoError(t, err)
 
-	data.waitForServiceRealized(defaultTimeout, svc.Name)
-	data.waitForServiceRealized(defaultTimeout, svc_lb.Name)
+	// Wait to make sure the Serivce is realized.
+	time.Sleep(3 * time.Second)
```

Comment:
Same here, tried with `testFromNode`, find out the test becomes unstable. With 20% of chance to report error in line 295.